server/main.lua

```diff
@@ -109,11 +109,16 @@ RegisterNetEvent('hospital:server:SetLaststandStatus', function(bool)
 end)
 
 RegisterNetEvent('hospital:server:SetArmor', function(amount)
-	local src = source
-	local Player = QBCore.Functions.GetPlayer(src)
-	if Player then
-		Player.Functions.SetMetaData("armor", amount)
-	end
+    local src = source
+    local Player = QBCore.Functions.GetPlayer(src)
+    if Player then
```

Comment:
Please change this if statement to `if not Player then return end` to prevent unnecessary indentation